gui/src/advancedDynamicTexture.ts

```diff
@@ -529,6 +529,9 @@ module BABYLON.GUI {
             return result;
         }
 
+		/**
+		 * FullScreenUI is created in a layer. This allows texture.layer.layerMask to be set.

```

Comment:
This requires a lot more explanation than this sentence.
Also, would be great to have the parameters documented.

Reply:
I'll add as much as I understand. That may be a shockingly small amount though. So far I have been trying to add the things I wish I had been able to find while working on my project.